android/app/src/main/assets/microlocations

```diff
@@ -1,122 +1,69 @@
-[
+{
```

Comment:
This is incorrect. The API and JSON have a different format. We want to use this format for JSON https://github.com/fossasia/open-event/wiki/%5BNextgen%5D-Open-Event-Universal-Format-Specification